src/core/energy_storage/electric_storage.jl

```diff
@@ -169,10 +172,13 @@ end
     can_grid_charge::Bool = off_grid_flag ? false : true
     installed_cost_per_kw::Real = 910.0
     installed_cost_per_kwh::Real = 455.0
+    installed_cost_constant::Real = 0.0
```

Comment:
@toddleif could you please add a brief description of this input to this markdown area?

Reply:
That's a great idea. I just added a description.

Reply:
@toddleif I just modified the description to be in-line with the inputs and to just give a short high-level description of what that input is. Could you review?